### DISCUSSION TITLE
【委托】显卡和内存条

### DISCUSSION THREAD

**Discussion by JobNumber-9527** [share121/inter-knot#2694] [其他]
有人知道显卡和内存条分别是处理什么的吗，玩个CS2只剩20帧了该换哪个<img src="https://img-static.mihoyo.com/communityweb/upload/ee7bb800b0c90f66fa9e321cb6d8ee17.png" alt="抱大腿" width="75" >

**Comment by jsishsjsj** [comment]
<img src="https://upload-bbs.miyoushe.com/upload/2023/03/09/3173bc3e69fcfebecfda741340c9ad22_6841250310756275678.png" alt="开拓者 吃瓜ing" width="75" >
1. **显卡（GPU）：**
   - 主要负责处理图像和视频的渲染工作。尤其是在玩游戏时，显卡承担了大部分的图形处理任务。更强大的显卡通常能提供更高的帧率、更好的图形质量和更流畅的游戏体验。

2. **内存条（RAM）：**
   - 主要负责暂时存储正在使用的数据和程序，以便CPU快速访问。充足的内存可以确保系统在运行多个程序或处理大数据量时保持流畅。但如果内存不足，系统可能会变得卡顿，尤其是在加载大型游戏场景时。

根据官方信息，《CS2》（《反恐精英2》）的最低配置要求如下：

- **操作系统：** Windows 10 64位
- **处理器：** 四核CPU，时钟频率至少为 2.5 GHz（例如 Intel Core i5-2500K 或 AMD FX-6300）
- **内存：** 8 GB RAM
- **显卡：** 支持 DirectX 11 的显卡，至少具有 2 GB VRAM（例如 NVIDIA GTX 660 或 AMD Radeon HD 7850）
- **存储空间：** 需要 85 GB 可用硬盘空间
- **网络：** 宽带互联网连接

根据上面的信息及自己的个人实际情况来升级配置，另外显卡驱动也别忘记升级呦～<img src="https://upload-bbs.miyoushe.com/upload/2023/01/18/12f2f4bd9867277637f01c0170d7e29c_3437575394729646264.png" alt="米游姬 献花" width="75" >

**Reply by JobNumber-9527** [reply]
谢谢<img src="https://img-static.mihoyo.com/communityweb/upload/fe198bb1523528c2b852d28626379a2a.png" alt="心情简单" width="75" >

**Comment by Bailinghai** [comment]
换个显卡

**Reply by scrooge-0** [reply]
> 换个显卡

cs2吃CPU，不要老想着显卡

**Reply by WFGoose** [reply]
回楼上，都20帧了很难不考虑显卡吧，当然CPU估计也不够用了就是说，这边建议直接换电脑呢

**Reply by scrooge-0** [reply]
同意，我只是说CPU也很关键，话没说清，抱歉，
但这个大哥有没有可能连核显了，不然是真该换电脑了【捂脸】

**Reply by JobNumber-9527** [reply]
用了5年左右的GTX1060<img src="https://upload-bbs.miyoushe.com/upload/2023/03/09/3173bc3e69fcfebecfda741340c9ad22_6841250310756275678.png" alt="开拓者 吃瓜ing" width="75" >

**Reply by scrooge-0** [reply]
啊，好嘛，50系出的时候可以换一波了
